ts/src/test/Exchange/test.fetchBorrowInterest.ts

```diff
@@ -1,17 +1,12 @@
-'use strict'
```

Comment:
@frosty00 @kroitor for most of the files, git successfully detected the rename but for a small % of them did not I don't know why (mainly tiny files) 
![image](https://user-images.githubusercontent.com/43336371/212165124-e9b4c843-d95e-4473-9f8e-fd94b044d184.png)

Anyway, I'm not sure there's much we can do about it